domains/certificates/Query.API/API.IntegrationTests/Factories/QueryApiWebApplicationFactory.cs

```diff
@@ -171,4 +190,14 @@ protected override IHost CreateHost(IHostBuilder builder)
     }
 
     public void Start() => Server.Should().NotBeNull();
+
+    protected override void Dispose(bool disposing)
+    {
+        foreach (var disposableChannel in disposableChannels)
+        {
+            disposableChannel.Dispose();
```

Comment:
Could this be fixed using `using`? so we wont have to implement the Dispose method

Reply:
This is for disposing gRPC channels. I don't see how